libfwupd/fwupd-self-test.c

```diff
@@ -43,6 +43,59 @@ fu_test_compare_lines (const gchar *txt1, const gchar *txt2, GError **error)
 	return FALSE;
 }
 
+static guint
+fu_common_string_replace (GString *string, const gchar *search, const gchar *replace)
```

Comment:
`_g_string_replace`? I'm still hopeful this will arrive in GLib one day.

Reply:
That's good yeah. I would say if you think this might land in glib someday then link to the bug or MR tracking it in a comment (file one if it doesn't yet exist)